fix run_exports

ocefpaf opened the pull request:
Same as #22 for `1.9.0`. I'll remove the other build numbers after this one is merged and published.

ocefpaf committed:
fix run_exports

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

hobu reviewed (commented):
Did you mean to revert the version to 1.9.0 here?

ocefpaf commented:
> Did you mean to revert the version to 1.9.0 here?

Yes. But I moved the 1.9.3 to the broken label and then back after we added the `run_exports` and pinning scheme for it.
